README.md

```diff
@@ -34,7 +34,8 @@ This template comes with:
 1. Install PostgreSQL in case you don't have it
```

Comment:
maybe somewhere in the README we need to mention ESBuild, that's used for the admin site, and that?

Reply:
sure